### PR TITLE
Add JAVA_HOME environment variable

### PR DESCRIPTION
**Pull request by mattnelson**
*Issue #, if available:*

*Description of changes:*

Add JAVA_HOME environment variable

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by EricEdens** [comment]
Matt, thanks for the PR. I'm queuing up a change that adds [container-structure-test](https://github.com/GoogleContainerTools/container-structure-test) to the project. Once that's pushed (hopefully today or tomorrow), can you add a test that verifies your environment variable?

**Comment by EricEdens** [comment]
All ready! Can you add a test to [test-image.yaml](https://github.com/corretto/corretto-8-docker/blob/master/test-image.yaml) then rebase and squash your PR?

**Comment by mattnelson** [comment]
@EricEdens Added tests. Also enhanced the test runner to work with both supported operating systems (linux/osx).

File: test-image.sh [hunk 1]
```diff
@@ -4,16 +4,26 @@ set -euo pipefail
...
 # For more info and usage examples, see:
 #   https://github.com/GoogleContainerTools/container-structure-test
+
+# only linux and osx are supported
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
+  cst_ostype="linux"
+elif [[ "$OSTYPE" == "darwin"* ]]; then
+  cst_ostype="darwin"
+else
+	echo "Unsupported operating system[$OSTYPE]"
```
Comment (EricEdens) [inline test-image.sh]: nit: two spaces here and below instead of a tab

**Review by EricEdens — COMMENTED** [review]
Looks good, and thanks for adding support to run on Mac! Last thing -- just rebase and squash down to a single commit.

**Review by EricEdens — APPROVED** [review]
lgtm!

**Pull request merged**